Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml

```diff
@@ -95,13 +88,15 @@
   </add-node>
   <attr path="/api/package/class[implements[@name='java.util.Collection']]/method[@name='iterator' and count(parameter)=0]" name="managedReturn">Java.Util.IIterator</attr>
   <attr path="/api/package/class[implements[@name='java.util.Iterator']]/method[@name='next' and count(parameter)=0]" name="managedReturn">Java.Lang.Object</attr>
+  <attr path="/api/package/class[implements[@name='java.util.Iterator']]/method[starts-with(@name,'next-') and count(parameter)=0]" name="return">java.lang.Object</attr>
   <attr path="/api/package/class[implements[@name='java.util.Map']]/method[@name='entrySet' and count(parameter)=0]" name="return">java.util.Set</attr>
 
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='CharRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='char']]/parameter[1]" name="managedType">Java.Lang.Character</attr>
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='IntRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='int']]/parameter[1]" name="managedType">Java.Lang.Integer</attr>
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='LongRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='long']]/parameter[1]" name="managedType">Java.Lang.Long</attr>
-  <attr path="/api/package[@name='kotlin.ranges']/class[@name='UIntRange']/method[starts-with(@name,'contains-') and count(parameter)=1 and parameter[1][@type='uint']]/parameter[1]" name="managedType">Kotlin.UInt</attr>
-  <attr path="/api/package[@name='kotlin.ranges']/class[@name='ULongRange']/method[starts-with(@name,'contains-') and count(parameter)=1 and parameter[1][@type='ulong']]/parameter[1]" name="managedType">Kotlin.ULong</attr>
```

Comment:
With the new generator, `UInt` and `ULong` are no longer a thing. The types exist, but there is no way to get one of them. Everything is the nice `uint` and `ulong`.